diff -up ./absl/base/internal/sysinfo_test.cc.armv7 ./absl/base/internal/sysinfo_test.cc
--- ./absl/base/internal/sysinfo_test.cc.armv7	2021-06-09 21:54:44.503755718 -0400
+++ ./absl/base/internal/sysinfo_test.cc	2021-06-09 22:56:14.459611843 -0400
@@ -43,13 +43,13 @@ TEST(SysinfoTest, NumCPUs) {
 // POWER is particularly problematic here; some Linux kernels expose the CPU
 // frequency, while others do not. Since we can't predict a priori what a given
 // machine is going to do, just disable this test on POWER on Linux.
-#if !(defined(__linux) && (defined(__ppc64__) || defined(__PPC64__)))
+#if !(defined(__linux) && (defined(__ppc64__) || defined(__PPC64__) || defined(__aarch64__) || defined(__arm__)))
 TEST(SysinfoTest, NominalCPUFrequency) {
   // Linux only exposes the CPU frequency on certain architectures, and
   // Emscripten doesn't expose it at all.
 #if defined(__linux__) &&                                                  \
         (defined(__aarch64__) || defined(__hppa__) || defined(__mips__) || \
-         defined(__riscv) || defined(__s390x__)) ||                        \
+         defined(__riscv) || defined(__s390x__))                     ||    \
     defined(__EMSCRIPTEN__)
   EXPECT_EQ(NominalCPUFrequency(), 1.0)
       << "CPU frequency detection was fixed! Please update unittest.";
